packages/bootleg-app-contracts/migrations/2_deploy_and_mint.js

```diff
@@ -1,17 +1,28 @@
 const BootlegToken = artifacts.require('BootlegToken');
+const BootlegTraderApp = artifacts.require('BootlegTraderApp');
 
-module.exports = async deployer => {
-  const tokenId = web3.utils.toBN(1);
+module.exports = async (deployer, network, accounts) => {
+  let firstOwner = web3.utils.toHex(0);
+  const metaDataURI = 'https://ipfs.infura.io/ipfs/QmSomeHash';
 
-  const accounts = await web3.eth.getAccounts();
+  // TOKEN NUMERO UNO!!!
+  const tokenId = web3.utils.toBN(1);
 
+  if (network == 'rinkeby') {
+    firstOwner = '0x61D87e60B3893801DbC9056c10bc124251Ea7980';
```

Comment:
This is your address? 

My understanding is that if you're using the truffle wallet to do the deploys like we are `accounts[0]` should have your address in it. 

Seems redundant or unexpected to me.

Is this address maybe not `accounts[0]` for you on rinkeby?

Reply:
Yeah, it's my address. I was not aware that's how the HD wallet worked. I'll remove this.